var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -49,23 +48,40 @@ class Podio(CMakePackage):
 
     def cmake_args(self):
         args = [
-            self.define('BUILD_TESTING', self.run_tests),
             self.define_from_variant('ENABLE_SIO', 'sio')
         ]
         return args
 
+    def setup_run_environment(self, env):
+        env.prepend_path('PYTHONPATH', self.prefix.python)
```

Comment:
Wouldn't this be done automatically if the package extends `python`?

Reply:
Hmm, that would be nicer indeed, but it's not quite the same thing, if I use `extends('python')` I get 

`/opt/spack/linux-ubuntu20.04-skylake/gcc-9.3.0/podio-0.12.0-in6htvlg5ayrm2o4fxkns5oj2n6a5dkp/lib64/python3.7/site-packages`

in my pythonpath, not `/opt/spack/linux-ubuntu20.04-skylake/gcc-9.3.0/podio-0.12.0-in6htvlg5ayrm2o4fxkns5oj2n6a5dkp/python/`, where the packages installs its files.

I know, it's a non-standard location, I'll try to change it for the next releases.